provider/resources.go

```diff
@@ -1070,12 +1066,6 @@ func ProviderFromMeta(metaInfo *tfbridge.MetadataInfo) *tfbridge.ProviderInfo {
 						// Explicitly map tag => tags to avoid confusion with tags => tagsCollection below.
 						Name: "tags",
 					},
-					"tags": {
-						// Conflicts with the pluralized `tag` property, which is the more strongly typed option for
-						// providing tags.  We keep this dynamically typed collection of tags as an option as well, but
-						// give it a different name.
-						Name: "tagsCollection",
-					},
```

Comment:
Why was this removed? I don't see a corresponding removal in upstream's release notes.

Reply:
I believe this was removed a while back: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/autoscaling_group
- there is no `tags` field. We could also remove the (TF)`tag` -> (pulumi)`tags` mapping but I imagine that would break users.

Reply:
update: like the other removals, this was removed for upstream v5. https://github.com/hashicorp/terraform-provider-aws/releases/tag/v5.0.0